react/features/conference/components/Conference.web.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+
+/**
+ * For legacy reasons, inline style for display none.
+ * @type {{display: string}}
+ */
+const displayNoneStyle = {
+    display: 'none'
+};
+
+/**
+ * Implements a React Component which renders initial conference layout
+ *
+ * @returns { ReactElement }
+ */
+const Conference = () =>
```

Comment:
Could I please have the React components defined as `class` like the previous/existing ones rather than as plain functions? I understand that they look simple as plain functions at the moment but (1) they are different from the previous/existing ones thus creating inconsistency, (2) more importantly, we'll have a harder time later on when/if we decide to add more methods to them, and (3) we want propTypes on them.